ng-dev/release/publish/actions/cut-stable.ts

```diff
@@ -19,16 +19,19 @@ import {ExternalCommands} from '../external-commands.js';
  */
 export class CutStableAction extends ReleaseAction {
   private _newVersion = this._computeNewVersion();
+  private _isNewMajor = this.active.releaseCandidate!.isMajor;
```

Comment:
```suggestion
  private _isNewMajor = this.active.releaseCandidate?.isMajor;
```


We can't assert that releaseCandidate will be defined right?